packages/cozy-sharing/src/components/Recipient.jsx

```diff
@@ -61,18 +61,29 @@ export const RecipientsAvatars = ({
       {reversedRecipients
         .slice(0, MAX_DISPLAYED_RECIPIENTS)
         .map((recipient, idx) => (
-          <Avatar
+          <RecipientAvatar
             key={idx}
-            text={Contact.getInitials(recipient)}
+            recipient={recipient}
             size={size}
-            textId={Contact.getDisplayName(recipient)}
             className={classNames(styles['recipient-avatar'])}
           />
         ))}
     </div>
   )
 }
 
+export const RecipientAvatar = ({ recipient, ...rest }) => {
+  const client = useClient()
+  return (
+    <Avatar
+      image={`${client.options.uri}${recipient.avatarPath}`}
+      text={Contact.getInitials(recipient)}
+      textId={Contact.getDisplayName(recipient)}
+      {...rest}
+    />
+  )
+}
```

Comment:
What does that mean if we're in an offline mobile app? Should we have a fallback?

Reply:
The  `Avatar` component handles the fallback automatically, since we still pass the text etc.

Reply:
Nop. 

https://github.com/cozy/cozy-ui/blob/master/react/Avatar/index.jsx#L53-L56

We pass an image, so there is no way Avatar will use the text?

Reply:
Shit you're right, I mistested that. The actual result isn't too bad:

<img width="314" alt="Capture d’écran 2020-09-08 à 14 38 17" src="https://user-images.githubusercontent.com/2261445/92477469-f5fa2080-f1e0-11ea-9dc5-c14ef246742c.png">

What do you think of testing with `navigator.onLine` whether we pass the image or not?

Reply:
> What do you think of testing with navigator.onLine whether we pass the image or not?

I don't know... Since we should have a cache one day... I think I'm wondering if the actual result is not enough